src/helpers/api/call.js

```diff
@@ -21,3 +21,17 @@ export const postRequest = async (route, data) => fetch(endpoint + route, {
   },
   body: JSON.stringify(data),
 }).then((response) => response);
+
+export const getOneDoctor = (id) => {
+  const resultFetch = fetch(`${endpoint}doctors/${id}`, {
+    method: 'GET',
+    mode: 'cors',
+    // headers: {
+    //   'Content-Type': 'application/json',
+    //   Authorization: `Bearer ${StorageManager.getToken()}`,
+    // },
```

Comment:
Amazing work so far!!

It would be nice to send authorization token to with the request and cross check it as well in the back end, this would ensure security of the endpoint. This way users that are not logged in cannot access the details of one doctor